build.gradle.kts

```diff
@@ -15,8 +17,9 @@ buildscript {
         resolutionStrategy {
             force("com.google.protobuf:protobuf-java:3.25.5")
             force("com.google.android.gms:play-services-basement:18.4.0")
-            force("io.netty:netty-codec-http2:4.1.108.Final")
+            force("io.netty:netty-codec-http2:4.1.115.Final")
             force("com.fasterxml.jackson.core:jackson-core:2.17.2")
+            force("commons-io:commons-io:2.18.0")
```

Comment:
are these used in production code?

Reply:
sort of. when running tests. these components
com.android.tools.utp:android-test-plugin-host-retention:31.7.3
com.android.tools.utp:android-test-plugin-host-emulator-control:31.7.3